### ISSUE TITLE
Add media favorites capability

### ISSUE THREAD

**Issue by marktaipan-cvisionai** [cvisionai/tator#1521]
**Category**
Feature Request

**Problem**
Users can't create media type attribute favorites

**Request**
- Allow backend to save media-based favorites
- Associated with #1520

**Comment by bctcvai** [comment]
With the new PR in-bound; one can add media_ids directly to a section. One could implement this PR by making a section called "Favorites" and adding media_id to it. One could make "Other favorites" Section too. Suggest we collapse/close this once https://github.com/cvisionai/tator/pull/1539 is merged.
